src/ArborX_Version.hpp.in

```diff
@@ -16,6 +16,9 @@
 
 #include <string>
 
+#cmakedefine ARBORX_VERSION_STRING "@ARBORX_VERSION_STRING@"
+#cmakedefine ARBORX_GIT_COMMIT_HASH "@ARBORX_GIT_COMMIT_HASH@"
```

Comment:
Are these names compatible with xSDK guidelines or we can pick what we want?

Reply:
```
M8.​​ Each package must provide ​ a runtime API​​ , for example a function call, to return the ​ current
version number​​ of the software and indicate what configure/CMAKE and compiler options were used
to build the package. For development versions of the software, each package must provide the
current commit ID in the repository. With this information users should be able to rebuild the package
in the same state. We do not currently require that version information for all dependent packages be
provided, so it may not be possible to rebuild the entire software stack in the same state.
```

For me, that reads like having functions that return the hash and the version is sifficient and there is no standardized name for these.

Reply:
xSDK does not specify any name conventions in the policies, except M1.